library/src/validations/maxLength/maxLength.test.ts

```diff
@@ -2,25 +2,55 @@ import { describe, expect, test } from 'vitest';
 import { maxLength } from './maxLength.ts';
 
 describe('maxLength', () => {
-  test('should pass only valid lengths', () => {
-    const validate = maxLength(3);
-
-    const value1 = '';
-    expect(validate._parse(value1).output).toBe(value1);
-    const value2 = '12';
-    expect(validate._parse(value2).output).toBe(value2);
-    const value3 = [1, 2, 3];
-    expect(validate._parse(value3).output).toBe(value3);
-
-    expect(validate._parse('1234').issues).toBeTruthy();
-    expect(validate._parse('123456789').issues).toBeTruthy();
-    expect(validate._parse([1, 2, 3, 4]).issues).toBeTruthy();
-    expect(validate._parse([1, 2, 3, 4, 5, 6, 7]).issues).toBeTruthy();
+  describe('should pass only valid lengths', () => {
+    test('empty string has valid length', () => {
+      const value1 = '';
+      expect(maxLength(3)._parse(value1).output).toBe(value1);
```

Comment:
Change you change that to how we implemented it at `email` with `.toEqual({ output: value })`? Same for the rest of the tests.